chapter_13_database_layer_validation.asciidoc

```diff
@@ -808,17 +865,6 @@ We should commit there:
 $ *git commit -am "Refactor list view to handle new item POSTs"*
 ----
 
-NOTE: So did I break the rule about never refactoring against failing tests?
-    In this case, it's allowed, because the refactor is required
-    to get our new functionality to work.
-    You should definitely never refactor against failing _unit_ tests.
-    But in my book it's OK for the FT for the current story you're working on
-    to be failing.footnote:[
-    If you really want a "clean" test run, you could add a skip or an early return
-    to the current FT, but you'd need to make sure you didn't accidentally forget it.]
```

Comment:
let's replace this with a `TIP` earlier on in the chapter

```
TIP: When working on a new feature, it's common to want to do a refactor of the application.
  Adding an early return to the FT you're currently working on will let you do that refactor against passing FTs
```

something like that!

We could maybe also add it to the recap at the end?

Reply:
Sounds good, how about just right above where we introduce the early return: "We’ve got some working code though, so let’s put an early return in the FT to separate (...)"?